docs/releases/2.2.txt

```diff
@@ -355,6 +355,12 @@ Miscellaneous
   dashes, e.g. ``550e8400-e29b-41d4-a716-446655440000`` instead of
   ``550e8400e29b41d4a716446655440000``.
 
+* ``ping_google`` management command now accepts two more optional arguments.
```

Comment:
"The ping_google management command now accepts the --site-domain and --use-http options. The first ... and the second ..."

That should be documented under "minor features" and only the backwards incompatible part about switch from http to https mentioned here.